README.md

```diff
@@ -326,6 +298,38 @@ You can configure your applications properties with environment variables, syste
 | okta.oauth2.issuer     | true | [Authorization Server](/docs/how-to/set-up-auth-server.html) issuer URL, i.e.: https://{yourOktaDomain}/oauth2/default |
 | okta.oauth2.clientId   | true | The Client Id of your Okta OIDC application |
 | okta.oauth2.clientSecret   | true | The Client Secret of your Okta OIDC application |
+| okta.oauth2.postLogoutRedirectUri | false | Set to a relative or absolute URI to enable [RP-Initiated (SSO) logout](https://developer.okta.com/blog/2020/03/27/spring-oidc-logout-options). |
+
+**NOTE**: On setting **postLogoutRedirectUri**, you will be redirected to it after the end of your session. Therefore, this resource must be available anonymously, so be sure to add it to your `HttpSecurity` configuration.
+
+<details>
```

Comment:
Moved the section above about the `postLogoutRedirectUri` prop into a collapsed section, as reader doesn't have enough context at this point (the full example is below).